### PR TITLE
Welcoming contact field

### PR DESCRIPTION
**Pull request by comradekingu**
## :scroll: Description

Welcoming and clear means of contact

## :bulb: Motivation and Context

No assumptions made, no immutable characteristics evoked, and no needless complexity.

## :green_heart: How did you test it?
Works in Weblate, Remmina, and other projects.

## :pencil: Checklist

- [ ] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] I updated the docs if needed
- [ ] All tests passing
- [x] No breaking changes

## :crystal_ball: Next steps
More contributions and more donations.

### CODE REVIEW

File: CONTACT.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Welcome to the Sanmill community.
+
+Settle differences by offering and accepting constrictive critique, within desired privacy.
```
Comment (calcitem) [inline CONTACT.md:3]: constrictive -> constructive

**Review by calcitem — CHANGES_REQUESTED** [review]
Thanks!

Find a typo.

**Comment by Leptopoda** [comment]
Why has the code of conduct been removed?

Also what is the benefit of the contact.md? I think conversations should be settled in the community tab and not closed behind email...

**Comment by comradekingu** [comment]
@Leptopoda
If you mean conflicts between people, then that would depend on the issue at hand, which is why I put in the part about privacy. Involving only those affected without bringing more attention than needed seems the path of least resistance.

The benefit of having CONTACT.md, and I suspect more-so getting rid of that other file is more donations (correlation in other projects) and more contributions.
It was introduced two months ago, and I am assuming it was not discussed then.
I have never come across a project where contributors are asked, or voluntarily pledge
allegiance to it. Consequently, the onus is on discussing its introduction, not its removal.

If you mean this thread, the discussion tab is the same as this without proposals attached (usually).
A thumbs up or down works here too, and I am answering my own question for discussion by proposing a solution.

In the very least this approach can be recited from memory, and there is some hope that it deescalates conflict rather than elevating it to either a maintainer that could be doing other things, or "community leaders" with less vested interest.

File: CONTACT.md (hunk 1 above)
Comment (comradekingu) [inline CONTACT.md:3]:
```suggestion
Settle differences by offering and accepting constructive critique, within desired privacy.
```

**Comment by comradekingu** [comment]
@Leptopoda Are you happy with the answer?
Was hoping to start working on this. :)

**Comment by calcitem** [comment]
I think it is OK **NOT** to add contact.md and remove the code of conduct.

Leptopoda is an experienced developer who can often make many constructive comments. For example, the previous https://github.com/calcitem/Sanmill/pull/368 proposed amendments to other documents. And including the migration of the text translation work of this project from the closed source POEditor to the open-source Weblate is also his suggestion. Although this three-day migration requires a lot of work, Weblate is indeed much more powerful, and this migration is worthwhile.

@comradekingu BTW, I saw that you translated Norsk, etc., on Weblate a few days ago. At that time, I was not familiar with the operation of Weblate and did not make an announcement. After finishing the reorganization, performing the reset operation, and synchronizing the repository, I found that you have disappeared from the contributor's record. I don't know why. Now that the project on Weblate has been configured, you can continue to translate if you wish. You partially translate the current arb file, but the record is missing. I set all Norsk entries to "untranslated" status. If the contributor confirms that the translation is OK later, he can modify the status by himself. Thanks a lot!

**Comment by Leptopoda** [comment]
I quite like having a code of conduct that actually covers far more cases and also offers contact i formation🤷‍♀️


I didn't reply because it's the decision of @calcitem and not mine. I already added my opinion.

Also Merry Christmas to those who celebrate

**Comment by calcitem** [comment]
Merry Christmas to those who celebrate too!

I hope everyone can play happily and enjoy fun on this project!

**Comment by comradekingu** [comment]
Merry Yuletide.

My culture is more important to me.

**Pull request closed without merging**